provision.yml

```diff
@@ -33,8 +33,12 @@
   roles:
     - { role: basic,         tags: ['common']            }
     - { role: apt,           tags: ['common','apt']      }
-    - { role: firewall,      tags: ['common','firewall'] }
-    - { role: ntp,           tags: ['common']            }
+    - role: firewall
+      tags: ['common','firewall']
+      when: ansible_virtualization_type != "docker"
+    - role: ntp
+      tags: ['common']
+      when: ansible_virtualization_type != "docker"
     - { role: users,         tags: ['common','users']    }
```

Comment:
In the latest master, these two roles don't do anything for docker containers, so we don't need docker-specific exceptions in the playbook.